.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 inherit_from:
-- https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop.yml
+- https://raw.githubusercontent.com/ManageIQ/manageiq/master/.rubocop_base.yml
```

Comment:
Am I right that we use the `.rubocop_base.yml` from ManageIQ/manageiq rather the one from the guides?